src/components/date/date.component.js

```diff
@@ -65,6 +65,9 @@ const DateInput = React.forwardRef(
       tooltipPosition,
       value,
       inputRef,
+      inputWidth,
+      labelWidth,
+      maxWidth,
```

Comment:
Currently the `maxWidth` prop is not documented in storybook because we have defined the `Textbox`-based props manually in the `stories.mdx` file.

Reply:
looks like this prop was excluded in our story, should be fixed now.